public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from 'react';
+import { useTable, Column } from 'react-table';
+import { Spinner, useStyles } from '@grafana/ui';
+import { getStyles } from './AlertsTable.styles';
+import { css } from 'emotion';
+import { logger } from '@percona/platform-core';
+import { AlertsService } from '../Alerts.service';
+import { Messages } from '../../../IntegratedAlerting.messages';
+import { Alert } from '../Alerts.types';
+import { formatAlerts } from './AlertsTable.utils';
+
+const { noData, columns } = Messages.alerts.table;
+
+const {
+  activeSince: activeSinceColumn,
+  labels: labelsColumn,
+  lastNotified: lastNotifiedColumn,
+  severity: severityColumn,
+  state: stateColumn,
+  summary: summaryColumn,
+} = columns;
+
+export const AlertsTable = () => {
+  const style = useStyles(getStyles);
+  const [pendingRequest, setPendingRequest] = useState(false);
+  const [data, setData] = useState<Alert[]>([]);
+
+  const getAlertRules = async () => {
+    setPendingRequest(true);
+    try {
+      const { alerts } = await AlertsService.list();
+      setData(formatAlerts(alerts));
+    } catch (e) {
+      logger.error(e);
+    } finally {
+      setPendingRequest(false);
+    }
+  };
+
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: summaryColumn,
+        accessor: 'summary',
+        width: '30%',
+      } as Column,
+      {
+        Header: severityColumn,
+        accessor: 'severity',
+        width: '5%',
+      } as Column,
+      {
+        Header: stateColumn,
+        accessor: 'status',
+        width: '5%',
+      } as Column,
+      {
+        Header: labelsColumn,
+        accessor: ({ labels }: Alert) => (
+          <div className={style.labelsWrapper}>
```

Comment:
nit: move this to a separate component?

Reply:
It's super small and doesn't affect readability too much, let's leave it as it is for now :)

Reply:
It's super small and doesn't affect readability too much, let's leave it as it is for now :)